src/core/models/model_list.py

```diff
@@ -0,0 +1,59 @@
+from typing import Callable, List, Dict, Type, Any
+from core.models.model import ActuatorModel, EnvironmentModel, Model, SensorModel
+from core.state import State
+from core.config import Config
+from utils.constants import ModelEnum
+
+
+class AttitudeDynamics(EnvironmentModel):
```

Comment:
These two classes should be moved to a separate dynamics-related file when they are ready to be implemented

Reply:
agreed

Reply:
This is on my TODO list in acs_model branch. I'm going to rip the EnvironmentModel class from this branch and bring it into mine to use as a template.